pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1273,6 +1278,14 @@ type ObjectZoneSpec struct {
 	DataPool PoolSpec `json:"dataPool"`
 }
 
+// RGWServiceSpec represent the spec for RGW service
+type RGWServiceSpec struct {
+	// The annotations-related configuration to add/set on each rgw service.
+	// nullable
+	// optional
+	Annotations *rookv1.Annotations `json:"annotations,omitempty"`
```

Comment:
`rookv1.Annotations` has a built-in nil value so the pointer is not needed here.